Uno-App/Source/Private/Match.cpp

```diff
@@ -17,13 +19,15 @@ void Match::StartNewMatch()
     }
 
     JoinPlayers(numPlayers);
+    CreateDeck();
+    SortCardsToPlayers();
 }
 
 void Match::JoinPlayers(const int& number)
 {
     for (int i=0; i < number; i++)
     {
-        JoinedPlayers.push_back(CreatePlayer(i));
+        JoinedPlayers.emplace_back(CreatePlayer(i));
```

Comment:
Why did you change this?

Reply:
I watched a video from Cherno and he said that `push_back` copy the value to the list and `emplace` add its reference directly without copying it. It's a performance improvement